src/main/java/polyglot/config/CommandLineArgs.java

```diff
@@ -89,6 +112,11 @@ private CommandLineArgs() {
   }
 
   private void initialize() {
+	// Short-circuit the preconditions checks if the "command" flag is present (as flag validation is command-specific) 
+	if (command().isPresent()) {
```

Comment:
Looks like this might be dangerous if we add more commands. Could we have a command-specific validation?